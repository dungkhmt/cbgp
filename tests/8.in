50 50
50 80
14 3
49 42
36 15
19 11
27 6
47 17
47 2
21 5
28 7
34 29
32 26
37 4
20 5
29 14
4 5
32 13
43 0
6 22
49 36
18 20
43 10
32 10
6 2
24 0
41 6
28 1
36 41
48 24
32 25
45 4
30 9
20 23
38 6
24 7
35 45
49 39
9 4
9 12
15 0
25 9
5 8
26 11
3 0
40 15
34 16
46 39
44 26
32 33
31 9
47 7
37 45
27 28
32 11
29 3
24 2
2 3
35 37
34 14
32 9
27 1
22 14
3 16
13 10
42 18
18 5
17 7
42 5
11 16
42 15
12 0
43 19
30 23
27 36
36 1
26 4
32 31
11 3
43 33
18 23
6 3
1
30 5 71 2 60 3 57 6 29 21 4 4 63 11 66 2 22 22 38 13 36 8 48 7 36 11 37 9 52 11 26 1 49 8 64 3 18 5 37 7 15 6 0 0 57 16 16 7 33 2 39 11 31 23 70 4 68 2 53 9 29 20 39 14 45 8 37 10 50 9 25 22 76 0 27 22 42 32 4 0 28 2 64 12 22 1 34 7 40 32 25 25 2 0 63 5 30 2 21 0