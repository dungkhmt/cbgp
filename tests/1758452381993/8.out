(14, 3), (49, 42), (36, 15), (19, 11), (27, 6), (47, 17), (47, 2), (21, 5), (28, 7), (34, 29), (32, 26), (37, 4), (20, 5), (29, 14), (4, 5), (32, 13), (43, 0), (6, 22), (49, 36), (18, 20), (43, 10), (32, 10), (6, 2), (24, 0), (41, 6), (28, 1), (36, 41), (48, 24), (32, 25), (45, 4), (30, 9), (20, 23), (38, 6), (24, 7), (35, 45), (49, 39), (9, 4), (9, 12), (15, 0), (25, 9), (5, 8), (26, 11), (3, 0), (40, 15), (34, 16), (46, 39), (44, 26), (32, 33), (31, 9), (47, 7), (37, 45), (27, 28), (32, 11), (29, 3), (24, 2), (2, 3), (35, 37), (34, 14), (32, 9), (27, 1), (22, 14), (3, 16), (13, 10), (42, 18), (18, 5), (17, 7), (42, 5), (11, 16), (42, 15), (12, 0), (43, 19), (30, 23), (27, 36), (36, 1), (26, 4), (32, 31), (11, 3), (43, 33), (18, 23), (6, 3), 
numberIntersection=0 
minDistance=3.0 
minAngle=0.05203631480921622 
minEdgeLength=4.242640687119285 
sumAngle=55.548727992269185 
sumEdgeLength=2502.9785314456844 
sumDistance=233257.74362544474 

0.0000000000 3.0000000000 0.0520363148 4.2426406871 55.5487279923 2502.9785314457 233257.7436254447
0: (142, 175),
1: (64, 196),
2: (81, 178),
3: (84, 174),
4: (137, 115),
5: (190, 179),
6: (66, 150),
7: (82, 195),
8: (141, 0),
9: (123, 144),
10: (129, 168),
11: (102, 171),
12: (128, 152),
13: (126, 165),
14: (86, 146),
15: (141, 195),
16: (99, 168),
17: (79, 188),
18: (152, 178),
19: (128, 171),
20: (169, 169),
21: (200, 199),
22: (76, 148),
23: (168, 162),
24: (125, 185),
25: (119, 155),
26: (116, 114),
27: (57, 176),
28: (71, 192),
29: (94, 163),
30: (128, 146),
31: (115, 150),
32: (106, 168),
33: (128, 161),
34: (101, 162),
35: (151, 115),
36: (45, 200),
37: (144, 113),
38: (0, 0),
39: (191, 198),
40: (133, 187),
41: (61, 155),
42: (156, 193),
43: (135, 171),
44: (4, 0),
45: (144, 23),
46: (196, 198),
47: (64, 179),
48: (138, 183),
49: (156, 200),
142 175 64 196 81 178 84 174 137 115 190 179 66 150 82 195 141 0 123 144 129 168 102 171 128 152 126 165 86 146 141 195 99 168 79 188 152 178 128 171 169 169 200 199 76 148 168 162 125 185 119 155 116 114 57 176 71 192 94 163 128 146 115 150 106 168 128 161 101 162 151 115 45 200 144 113 0 0 191 198 133 187 61 155 156 193 135 171 4 0 144 23 196 198 64 179 138 183 156 200 
0: (142, 175), 1: (64, 196), 2: (81, 178), 3: (84, 174), 4: (137, 115), 5: (190, 179), 6: (66, 150), 7: (82, 195), 8: (141, 0), 9: (123, 144), 10: (129, 168), 11: (102, 171), 12: (128, 152), 13: (126, 165), 14: (86, 146), 15: (141, 195), 16: (99, 168), 17: (79, 188), 18: (152, 178), 19: (128, 171), 20: (169, 169), 21: (200, 199), 22: (76, 148), 23: (168, 162), 24: (125, 185), 25: (119, 155), 26: (116, 114), 27: (57, 176), 28: (71, 192), 29: (94, 163), 30: (128, 146), 31: (115, 150), 32: (106, 168), 33: (128, 161), 34: (101, 162), 35: (151, 115), 36: (45, 200), 37: (144, 113), 38: (0, 0), 39: (191, 198), 40: (133, 187), 41: (61, 155), 42: (156, 193), 43: (135, 171), 44: (4, 0), 45: (144, 23), 46: (196, 198), 47: (64, 179), 48: (138, 183), 49: (156, 200), 
numberIntersection=0 minAngle=0.05203631480921622 sumAngle=55.54872799227399 minEdgeLength=4.242640687119285 sumEdgeLength=2502.9785314457044 minDistance=3.0 sumDistance=233257.74362543473 
Time taken: 43.700 s
